tutorial/4_hextof_workflow.ipynb

```diff
@@ -46,14 +53,33 @@
    "source": [
     "local_path = Path(sed.__file__).parent.parent/'tutorial/'\n",
     "config_file = local_path/'hextof_config.yaml'\n",
-    "assert config_file.exists()"
+    "assert config_file.exists()\n",
+    "config_override = {\n",
+    "    \"core\": {\n",
+    "        \"paths\": {\n",
+    "            \"data_raw_dir\": \"/asap3/flash/gpfs/pg2/2023/data/11019101/raw/hdf/offline/fl1user3\", \n",
+    "            \"data_parquet_dir\": \"/home/agustsss/temp/sed_parquet/\",\n",
```

Comment:
This should not point to your home folder. Maybe let's use some relative path, assuming the sed package is installed in the home folder, and this file resides in the tutorials